components/lcd/esp_lcd_gc9a01/include/esp_lcd_gc9a01.h

```diff
@@ -66,12 +66,19 @@ esp_err_t esp_lcd_new_panel_gc9a01(const esp_lcd_panel_io_handle_t io, const esp
  */
 #define GC9A01_PANEL_BUS_SPI_CONFIG(sclk, mosi, max_trans_sz)   \
     {                                                           \
-        .sclk_io_num = sclk,                                    \
         .mosi_io_num = mosi,                                    \
-        .miso_io_num = -1,                                      \
-        .quadhd_io_num = -1,                                    \
+        .miso_io_num = 0,                                       \
```

Comment:
Setting `miso_io_num` to 0 would assign it to GPIO0. Can you change it back to -1? Same issue on lines below